README.md

```diff
@@ -1,7 +1,25 @@
 # <img src="docs/img/sonobuoy-logo.png" width="400px" > [![Build Status][status]][travis]
 
-[status]: https://travis-ci.org/heptio/sonobuoy.svg?branch=master
-[travis]: https://travis-ci.org/heptio/sonobuoy/#
+- [Overview](#overviewoview)
```

Comment:
I think the TOC might be too much for a front-page readme. We should get to the point and explain what Sonobuoy is, how it works, and maybe a quick, getting started. I think anything more than that we should move off to other docs pages and dig in deeper.

Reply:
TOC removed and readme made shorter